packages/code-du-travail-nlp/api/ready.py

```diff
@@ -3,7 +3,10 @@
 
 
 def add_ready(app, nlp):
-    @app.route('/api/ready', methods=['GET'])
+    @app.route("/api/ready", methods=["GET"])
     @cross_origin()
     def ready():  # pylint: disable=unused-variable
-        return jsonify({'ready': nlp.is_ready, 'what': nlp.what})
+        if nlp.is_ready:
+            return "green"
+        else:
+            abort(500, "Service unavailable")
```

Comment:
maybe status 503 | Service Unavailable | Service temporairement indisponible ou en maintenance.

Reply:
thanks... I'm not very fluent in REST :p